pandas/core/index.py

```diff
@@ -19,7 +19,7 @@
     ensure_index_from_sequences,
     get_objs_combined_axis,
 )
-from pandas.core.indexes.multi import _sparsify  # noqa:F401
+from pandas.core.indexes.multi import sparsify_labels  # noqa:F401
```

Comment:
actually i think you can just remove this line entirely (its not used inside)

Reply:
i mean, nothing is used in this file

Reply:
ok, then wonder if we need this *at* all here (and maybe even re-evaluate *all* of these exports); but let's do later.

Reply:
i think the idea is to remove the file in 2.0